gf-webapp/src/logic/rooms/roomsPage/RoomsPage.tsx

```diff
@@ -4,20 +4,30 @@ import {RoomsPromo} from "src/logic/rooms/roomsPage/roomsPromo/RoomsPromo";
 import {Navigation} from "src/logic/rooms/roomsPage/navigation/Navigation";
 import {ServicesBlock} from "src/component/servicesBlock/ServicesBlock";
 import {BookingBlock} from "src/component/bookBlock/BookingBlock";
+import {useState} from "react";
+import {Loader} from "gf-ui-lib/src/components/Loader/Loader";
 import styles from "src/logic/rooms/roomsPage/RoomsPage.module.scss";
 
 
 export const RoomsPage = () => {
   const {roomsPage} = useDictionary().dictionary;
+  const [isDataLoading, setIsDataLoading] = useState(false);
+
+  function handleDataLoading() {
+    setIsDataLoading(true);
+  }
 
   return (
     <div>
       <PageBorder>
+        <Loader isLoading={isDataLoading} />
         <RoomsPromo />
         <div className={styles.about}>
           {roomsPage.description}
         </div>
-        <Navigation />
+        <Navigation
+          handleDataLoading={handleDataLoading}
```

Comment:
const setIsNavigationBlockInitTrue = () => setIsNavigationBlockInit(true);

handleDataLoading={setIsNavigationBlockInitTrue}

Reply:
I prefer to use an arrow function directly in the component